lib/timeline/component/item/Item.js

```diff
@@ -23,7 +24,11 @@ class Item {
     this.data = data;
     this.dom = null;
     this.conversion = conversion || {};
-    this.options = options || {};
+    this.defaultOptions = {
+      locales,
+      locale: 'en'
+    };
+    this.options = util.extend({}, this.defaultOptions, options);
```

Comment:
Note: https://github.com/visjs/vis-timeline/issues/55

Reply:
What do you want me to modify this PR ?

Do you think `util.extend...` is performed in another PR?
I think we address it in this PR, since `this.options = util.extend({}, this.defaultOptions)` is a pattern widely used in vis-timeline already.
https://github.com/visjs/vis-timeline/blob/6a6899a125ae859d2eaa9e05e2d81c3cc826dc98/lib/timeline/component/CurrentTime.js#L30
https://github.com/visjs/vis-timeline/blob/6a6899a125ae859d2eaa9e05e2d81c3cc826dc98/lib/timeline/component/Legend.js#L33
and lots.

Reply:
ok